rezervo/chains/schema.py

```diff
@@ -0,0 +1,28 @@
+from pydantic import BaseModel
+
+from rezervo.providers.schema import Branch
+
+
+class ThemeSpecificImages(BaseModel):
+    largeLogo: str
+
+
+class ThemeAgnosticImages(BaseModel):
+    smallLogo: str
+
+
+class ChainProfileImages(BaseModel):
+    light: ThemeSpecificImages
+    dark: ThemeSpecificImages
+    common: ThemeAgnosticImages
+
+
+class ChainProfile(BaseModel):
+    identifier: str
+    name: str
+    images: ChainProfileImages
+
+
+class Chain(BaseModel):
```

Comment:
Should be renamed to avoid confusion with `Chain` from `rezervo/chains/chain.py`. I think the original `ChainResponse` is fine since it's actually only used for api responses, but something like `ChainDetails` works too.

Reply:
🐫